tools/httpstan/README.md

```diff
@@ -0,0 +1,52 @@
+# bucketeer-httpstan
+
+Currently, the calculator service is written in Python, using Bayesian Algorithm to calculate the experiment
+probabilities.\
+Due to the following reasons, we want to rewrite the service in Go and, for Bayesian analysis, use the `HttpStan`.\
+This directory is used to build the `HttpStan` Docker image.
+
+## How to build httpstan image
+
+If you have compiling errors during the image build process, you might need to increase the docker VM memory.
+
+```bash
+make docker-build
+```
+
+## How to run httpstan image
+
+```bash
+docker run -p 8080:8080 -it --rm --name httpstan bucketeer-httpstan:latest
```

Comment:
Because we set the `TAG` when we build, we should create another rule in the Make to run this command with the correct version. What do you think?

Reply:
You thoughts are the right way, I think we could create a command in Makefile to run HttpStan with the `TAG` .